addons/web/static/src/views/kanban/kanban_model.js

```diff
@@ -171,8 +172,16 @@ class KanbanGroup extends Group {
     }
 
     async load() {
-        const proms = [];
-        proms.push(super.load());
+        this.loadTooltip();
```

Comment:
I think there may be a concurrency issue here, what if the groupby changes before tooltips had a chance to come back ?
I think using a new instance of keeplast for tooltips is necessary